### ISSUE TITLE
decorator @given and @Given are different in 1.2.6?

### ISSUE THREAD

**Issue by alluir42** [behave/behave#673]
Hi,

I'm experimenting with behave 1.2.6 now, and, well, basically all my steps are gone. :(

```
from behave import *
@given(u'Hello world')
def hello_word(context):
   pass

```
is not recognized. However if I change @given to @ Given, it is suddenly recognized and matches my step in the feature file. Do I need to convert all step decorators? Or is there some magic option I'm missing?

```
       Scenario: hello
         Given hello world
         And hello world
```

**Comment by alluir42** [comment]
It is very weird, because in both cases, if I do a printout from the environment.py to list behave.step_registry.registry.steps; it shows me an empty step catalog. It is mysterious.

**Comment by jenisys** [comment]
@alluir42 
Both @given and @Given step decorators work the same in behave-1.2.6 (as before).
Otherwise, the behave's feature tests would not work.
See that test case above to ensure that wildcard imports of step decorators works as expected.

**Comment by alluir42** [comment]
The test case is the one above, that I copy-pasted :). There is a twist, that the py containing the step is not in the step folder, but imported by a file in the step folder. When I copied the file into the step folder, the step suddenly started working.

I have done some digging, found that behave.step_registry.registry.steps has been moved to behave.runner.the_step_registry . It is kind of confusing, because formatters still have access to it with the old name, so it took me a while to find this out. But when I finally could see from environment.py what steps behave sees (using the step registry) I could fix the importing too.

But anyway, when steps are in the step folder it is working any way, so I guess I will close it down. And I managed to fix my problem too with some trickery around importing the py modules. Thank you for your help! :)
